electron/app/recoil/selectors.ts

```diff
@@ -187,6 +216,111 @@ export const refreshColorMap = selector({
   },
 });
 
+export const isLabel = selectorFamily({
+  key: "isLabel",
+  get: (field) => ({ get }) => {
+    const types = get(labelTypes);
+    return Boolean(types[field]);
+  },
+});
+
+export const modalFieldIsFiltered = selectorFamily({
+  key: "modalFieldIsFiltered",
+  get: (field: string) => ({ get }): boolean => {
+    const label = get(isLabel(field));
+
+    if (!label) {
+      return false;
+    }
+
+    const range = get(atoms.modalFilterLabelConfidenceRange(field));
+    const bounds = get(labelConfidenceBounds(field));
+    const none = get(atoms.modalFilterLabelIncludeNoConfidence(field));
+    const include = get(atoms.modalFilterIncludeLabels(field));
+    const maxMin = label ? 0 : bounds[0];
+    const minMax = label ? 0 : bounds[1];
+    const stretchedBounds = [
+      maxMin < bounds[0] ? maxMin : bounds[0],
+      minMax > bounds[1] ? minMax : bounds[1],
+    ];
+
+    const rangeIsFiltered = stretchedBounds.some(
+      (b, i) => range[i] !== b && b !== null && range[i] !== null
+    );
+
+    return Boolean(include.length) || rangeIsFiltered || !none;
+  },
+});
+
+export const fieldIsFiltered = selectorFamily({
+  key: "fieldIsFiltered",
+  get: (field: string) => ({ get }): boolean => {
+    const label = get(isLabel(field));
+    const numeric = get(isNumericField(field));
+    const range = get(
+      label
+        ? atoms.filterLabelConfidenceRange(field)
+        : atoms.filterNumericFieldRange(field)
+    );
+    const bounds = get(
+      label ? labelConfidenceBounds(field) : numericFieldBounds(field)
+    );
+    const none = get(
+      label
+        ? atoms.filterLabelIncludeNoConfidence(field)
+        : atoms.filterNumericFieldIncludeNone(field)
+    );
+    const include = get(atoms.filterIncludeLabels(field));
+    const maxMin = label ? 0 : bounds[0];
+    const minMax = label ? 1 : bounds[1];
+    const stretchedBounds = [
+      maxMin < bounds[0] ? maxMin : bounds[0],
+      minMax > bounds[1] ? minMax : bounds[1],
+    ];
+
+    if (!label && !numeric) return false;
+
+    const rangeIsFiltered = stretchedBounds.some(
+      (b, i) => range[i] !== b && b !== null && range[i] !== null
+    );
+
+    if (numeric) return rangeIsFiltered || !none;
+
+    return Boolean(include.length) || rangeIsFiltered || !none;
+  },
+});
+
+export const labelConfidenceBounds = selectorFamily({
+  key: "labelConfidenceBounds",
+  get: (label) => ({ get }) => {
+    return get(atoms.stateDescription).view_stats.labels[label]
+      .confidence_bounds;
+  },
+});
+
+export const numericFieldBounds = selectorFamily({
+  key: "numericFieldBounds",
+  get: (label) => ({ get }) => {
+    const bounds = get(atoms.stateDescription).view_stats.numeric_field_bounds[
```

Comment:
Getting this in storybook:
`Cannot read property 'numeric_field_bounds' of undefined`

The issue is that fields of `stateDescription`, like `view_stats`, aren't always defined. This error isn't as likely to come up in the app, since the components that use this selector don't usually render when no dataset is loaded, but I think it can happen when unloading a dataset (maybe by killing the server). A couple earlier selectors in this file check whether `view_stats` is defined, so this and the `labelConfidenceBounds` selector above probably should too.